conversion/conversion.go

```diff
@@ -417,6 +417,70 @@ func getSeekable(f *os.File) (*os.File, int64, error) {
 	return fcopy, n, nil
 }
 
+func VerifyTargetDb(targetDb, project, instance, dbName string) (string, bool, error) {
+	ctx := context.Background()
+	adminClient, err := database.NewDatabaseAdminClient(ctx)
+	if err != nil {
+		return "", false, fmt.Errorf("can't create admin client: %w", analyzeError(err, project, instance))
+	}
+	defer adminClient.Close()
+
+	dbPath := fmt.Sprintf("projects/%s/instances/%s/databases/%s", project, instance, dbName)
+	_, err = adminClient.GetDatabase(ctx, &adminpb.GetDatabaseRequest{Name: dbPath})
+	if err != nil {
+		// Differentiate between "db does not exist" and other errors? Assuming err means db does not exist for now
+		return targetDb, true, nil
+	}
+	//if resp.DatabaseDialect == "POSTGRESQL" {
+	//	targetDb = TARGET_EXPERIMENTAL_POSTGRES
+	//} else {
+	//	targetDb = TARGET_SPANNER
+	//}
+	dbDdl, err := adminClient.GetDatabaseDdl(ctx, &adminpb.GetDatabaseDdlRequest{Database: dbPath})
+	if err != nil {
+		fmt.Printf("\nCan't fetch database ddl: %v\n", err)
+		return "", false, fmt.Errorf("can't fetch database ddl")
+	}
+	if len(dbDdl.Statements) != 0 {
+		fmt.Printf("\nWriting to existing databases with non-empty ddl not supported\n")
+		return "", false, fmt.Errorf("Writing to existing databases with non-empty ddl not supported")
```

Comment:
Consider dropping the double-negation and write this as "HarbourBridge only supports writing to existing databases with an empty schema."